drafts/Policy-Authentication.md

```diff
@@ -0,0 +1,53 @@
+# Authentication Policy
+This document details the policies around the various authentication mechanisms within the Status ecosystem. 
+
+NOTES:
+- a ~~markthrough~~ refers to a standards guideline that does not apply to Status
+
+## Memorized Secrets (Passwords)
+- Current (relevant) [NIST](https://pages.nist.gov/800-63-3/sp800-63b.html) Standards Guidelines
+  - memorized secret guidelines is [Section 5.1](https://pages.nist.gov/800-63-3/sp800-63b.html#sec5)
+  - SHALL be minimum 8 characters if chosen by user
+  - ~~SHALL be minimum 6 characters if chosen randomly by a Credential Service Provider (CSP) and MAY be entirely numeric~~
+    - ~~CSP SHALL use an approved [random bit generator](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-90Ar1.pdf)~~
+  - A verifier SHOULD allow for at least 64 characters in length
+  - All printable ASCII characters as well as space SHOULD be acceptable
+  - Unicode SHOULD be acceptable, each Unicode code should be counted as a single character
+  - If Unicode is accepted, a [normalization process](https://unicode.org/reports/tr15/) SHOULD be applied
+  - A verifier MAY replace multiple spaces as a single space
+  - a verifier SHALL NOT truncate secrets
+  - A verifier SHALL NOT prompt user to use specific types of information, _e.g._ "favorite pet" questions
+  - A verifier SHALL compare prospective secrets against [a list]() that contains values known to be commonly-used, expected, or compromised
+    - if found, a verifier SHALL advise chosing a different password
+  - a verifier SHOULD offer guidance to the user, such as [password strength meters](https://www.ndss-symposium.org/ndss2014/programme/very-weak-very-strong-analyzing-password-strength-meters/)
+  - A verifier SHALL implement rate-limiting mechanism to limit number of failed authentication attempts
+  - A verifier SHOULD NOT impose composition rules
+  - A verifier SHOULD NOT require regular password changes
+    - but SHOULD impose a password change if an indicator of compromise is detected
+  - A verifier SHOULD permit "paste" functionality for entering memorized secret
+  - A verifier SHOULD offer option to display the secret
+    - they MAY choose to show each character in cleartext as it is entered for a short time
```

Comment:
Imo this one is questionable w.r.t. shoulder surfing. I personally prefer being able to enter passwords while other people are potentially somewhat close by.

Reply:
a paste would never show, manual entry (particularly on mobile) MAY show the character for a short period.